devicetypes/smartthings/fibaro-heat-controller.src/fibaro-heat-controller.groovy

```diff
@@ -0,0 +1,326 @@
+/**
+ *	Copyright 2018 SmartThings
+ *
+ *	Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *	in compliance with the License. You may obtain a copy of the License at:
+ *
+ *		http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *	Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+
+ *	on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *	for the specific language governing permissions and limitations under the License.
+ *
+ */
+metadata {
+	definition (name: "Fibaro Heat Controller", namespace: "smartthings", author: "Samsung", mnmn: "SmartThings", vid: "SmartThings-smartthings-Z-Wave_Battery_Thermostat", ocfDeviceType: "oic.d.thermostat") {
+		capability "Thermostat Mode"
+		capability "Refresh"
+		capability "Battery"
+		capability "Thermostat Heating Setpoint"
+		capability "Health Check"
+		capability "Thermostat"
```

Comment:
We want `"Temperature Measurement"` here too and to duplicate the events to the parent and the child so that we can guarantee all the Thermostat plumbing will work.

Reply:
Hmm. Now that I think about this I don't know how this will work without the child sensor...

Put "temperature measurement" in and give it a "0" initial value I guess. I can already tell we're going to have trouble getting this integrated.